apps/cookbook/src/app/showcase/styling-list/styling-list-showcase.html

```diff
@@ -0,0 +1,120 @@
+<p>
+  Kirby offers default styling of ordered <code>&lt;ol&gt;</code> and unordered
+  <code>&lt;ul&gt;</code> lists (but not description lists <code>&lt;dl&gt;</code>).
+</p>
+<h2>Unordered List Example</h2>
+<ul>
+  <li>Aenean id odio turpis</li>
+  <li>Lorem ipsum dolor sit amet</li>
+  <li>Donec ullamcorper, risus id venenatis</li>
+</ul>
+<h2>Unordered List /w Nested Lists Example</h2>
+<ul>
+  <li>Maecenas purus sapien</li>
+  <li>Pellentesque ultricies mi</li>
+  <li>
+    Dignissim et lorem mollis
+    <ul>
+      <li>Duis eget tincidunt sapien</li>
+      <li>Phasellus viverra ipsum sed</li>
+      <li>
+        Donec pulvinar aliquet turpis
+        <ul>
+          <li>Praesent vitae augue sed</li>
+          <li>Vitae convallis enim lobortis</li>
+          <li>Nisi condimentum ultricies nec</li>
+        </ul>
+      </li>
+    </ul>
+  </li>
+</ul>
+<h2>Ordered List Example</h2>
+<ul>
+  <li>Nulla cursus sem nulla</li>
+  <li>Sed egestas lorem pharetra</li>
+  <li>Pellentesque habitant morbi</li>
+</ul>
+<h2>Ordered List /w Nested Lists Example</h2>
+<ol>
+  <li>Proin consectetur pharetra</li>
+  <li>Aenean lobortis porta dictum</li>
+  <li>
+    Vestibulum in tortor vulputate
+    <ol>
+      <li>Fusce iaculis ante ac diam</li>
+      <li>Ac elementum nisi eleifend</li>
+      <li>
+        Maecenas nec convallis orci
+        <ol>
+          <li>Praesent ut felis quis</li>
+          <li>Etiam quis placerat neque</li>
+          <li>Nunc malesuada luctus varius.</li>
+        </ol>
+      </li>
+    </ol>
+  </li>
+</ol>
+<h2>Mixed List</h2>
+<ul>
+  <li>Proin at pharetra ligula</li>
+  <li>Integer id diam eleifend ac</li>
+  <li>
+    Sit amet blandit tortor dignissim
+    <ol>
+      <li>Consectetur adipiscing elit</li>
+      <li>Mauris fringilla sodales nulla</li>
+      <li>
+        In pulvinar massa ac porttitor
+        <ul>
+          <li>Proin at pharetra ligula</li>
+          <li>Suscipit erat a, aliquet nisi</li>
+          <li>Curabitur tempus dolor massa</li>
+        </ul>
+      </li>
+    </ol>
+  </li>
+</ul>
+<h2>Text Overflow in List</h2>
+<ul>
+  <li>
+    Proin at pharetra ligula, donec ullamcorper, risus id venenatis. Integer id diam eleifend ac,
+    consectetur adipiscing elit. Fusce iaculis ante ac diam ac elementum nisi eleifend in pulvinar
+    massa ac porttitor.
+  </li>
+</ul>
```

Comment:
Instead of having duplicate markup here and in the example file, you could use the example directly
```suggestion
<cookbook-styling-list-example></cookbook-styling-list-example>
```